diff --git a/node_modules/react-native-worklets/android/build.gradle b/node_modules/react-native-worklets/android/build.gradle
index 61c565b..c5c7ac5 100644
--- a/node_modules/react-native-worklets/android/build.gradle
+++ b/node_modules/react-native-worklets/android/build.gradle
@@ -294,14 +294,10 @@ def validateReactNativeVersionResult = providers.exec {
 }
 
 task assertMinimalReactNativeVersionTask {
-    doFirst {
-        if (validateReactNativeVersionResult.getResult().get().exitValue != 0) {
-            throw new GradleException(validateReactNativeVersionResult.getStandardError().getAsText().get().trim())
-        }
-    }
+    // Patched: skip version check for RN 0.76 compatibility
 }
 
-preBuild.dependsOn(assertMinimalReactNativeVersionTask)
+// preBuild.dependsOn(assertMinimalReactNativeVersionTask)
 
 task assertNewArchitectureEnabledTask {
     onlyIf { !IS_NEW_ARCHITECTURE_ENABLED }
